sdk/mixedreality/azure-mixedreality-remoterendering/src/main/java/com/azure/mixedreality/remoterendering/RemoteRenderingAsyncClient.java

```diff
@@ -0,0 +1,535 @@
+        return new PollerFlux<>(
+            Duration.ofSeconds(10),
+            pollingContext -> impl.getRemoteRenderings().createSessionWithResponseAsync(accountId, sessionId, ModelTranslator.toGenerated(options), context).map(mapper),
+            pollingContext -> {
+                Mono<T> response = impl.getRemoteRenderings().getSessionWithResponseAsync(accountId, sessionId, context).map(mapper);
+                return response.map(session -> {
+                    final RenderingSessionStatus sessionStatus = statusgetter.apply(session);
+                    LongRunningOperationStatus lroStatus = LongRunningOperationStatus.NOT_STARTED;
+                    if (sessionStatus == RenderingSessionStatus.STARTING) {
+                        lroStatus = LongRunningOperationStatus.IN_PROGRESS;
+                    } else if (sessionStatus == RenderingSessionStatus.ERROR) {
+                        lroStatus = LongRunningOperationStatus.FAILED;
+                    } else if (sessionStatus == RenderingSessionStatus.READY) {
+                        lroStatus = LongRunningOperationStatus.SUCCESSFULLY_COMPLETED;
+                    } else if (sessionStatus == RenderingSessionStatus.STOPPED) {
+                        lroStatus = LongRunningOperationStatus.USER_CANCELLED;
+                    } else {
+                        lroStatus = LongRunningOperationStatus.FAILED;
+                    }
+                    return new PollResponse<>(lroStatus, session);
+                });
+            },
+            (pollingContext, pollResponse) -> impl.getRemoteRenderings().stopSessionWithResponseAsync(accountId, sessionId, context).then(Mono.just(pollingContext.getLatestResponse().getValue())),
+            pollingContext -> Mono.just(pollingContext.getLatestResponse().getValue())
+        );
+    }
+
+
+
+    /**
+     * Gets properties of a particular rendering session.
+     *
+     * @param sessionId An ID uniquely identifying the rendering session for the given account. The ID is case
+     *     sensitive, can contain any combination of alphanumeric characters including hyphens and underscores, and
+     *     cannot contain more than 256 characters.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the rendering session.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<RenderingSession> getSession(String sessionId) {
+        return impl.getRemoteRenderings().getSessionWithResponseAsync(accountId, sessionId, Context.NONE).map(s -> ModelTranslator.fromGenerated(s.getValue()));
```

Comment:
This should use `withContext` helper method to call the service with the Reactor subscriber context. Do the same in other methods too.